opc_ua/client.py

```diff
@@ -145,7 +148,11 @@ def CTNGenerate_C(self, buildpath, locations):
         with open(c_path, 'w') as c_file:
             c_file.write(c_code)
 
-        LDFLAGS = ['"' + os.path.join(Open62541LibraryPath, "libopen62541.a") + '"', '-lcrypto']
+        lib_crypto = f'"{os.path.join(MingwPathOptLibraryPath, "libcrypto.a")}"'  if os.name == 'nt' else  '-lcrypto'
+        lib_open62541 = f'"{os.path.join(Open62541LibraryPath, "libopen62541.a")}"'
+        LDFLAGS = [lib_open62541, lib_crypto]
```

Comment:
This is too specific to your particular windows installer layout. Please give a chance to beremiz_public_dist.